supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -428,6 +428,8 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 *See also*: xref:logical-topology[logical topology], xref:physical-topology[physical topology]
 
 // EAP: Added "In Red Hat JBoss Enterprise Application Platform,"
+
+*See also*:
```

Comment:
Remove, as there's already a "see also" for this entry.

Reply:
Done @mportman12